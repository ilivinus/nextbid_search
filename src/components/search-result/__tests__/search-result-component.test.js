import React from 'react';
import renderer from 'react-test-renderer';
import SearchResultComponent from '../search-result-component';

describe("#SearchResultComponent",() => {

    it("Should take the snapshot of the component",()=>{
        let component = renderer.create(
            <SearchResultComponent 
                items={[]} />)
        let tree =  component.toJSON();

        expect(tree).toMatchSnapshot();

    })

    it("Should call the appropriate prop",()=>{

        let data = ["Marnix","Livinus","Chidi","Peter"];

        let component = renderer.create(
            <SearchResultComponent 
                items={data} />)

        let tree = component.toJSON();
        expect(tree.children.length).toBe(4);
    })
});