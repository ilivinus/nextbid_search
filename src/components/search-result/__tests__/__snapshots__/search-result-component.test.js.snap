// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#SearchResultComponent Should take the snapshot of the component 1`] = `<div />`;
